libraries/BLE/src/BLEDeviceManager.h

```diff
@@ -0,0 +1,421 @@
+/*
+  BLE Device API
+  Copyright (c) 2016 Arduino LLC. All right reserved.
+
```

Comment:
Is this an internal class? If so, can we move it to `src/internal/BLEDeviceManager.h`?